src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/DeploymentOptions.cs

```diff
@@ -2,7 +2,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
+using System;
```

Comment:
nit : do you want to move this file to a common folder of sorts?

Reply:
Yeah, that thought crossed my mind but I don't see one already established.  Think it's worth making a new Common folder?  The other possibility is to put it with the DacFxService folder since SchemaCompareService is built on DacFx so the hierarchy makes a little more sense that way.

Reply:
yeah i dot think there is one already,  am okay with how you want arrange this.